src/transformers/quantizers/quantizer_torchao.py

```diff
@@ -91,6 +92,11 @@ def _linear_extra_repr(self):
     SUPPORTED_SAFE_SERIALIZATION_CONFIGS = [
         torchao.quantization.Float8WeightOnlyConfig,
         torchao.quantization.Float8DynamicActivationFloat8WeightConfig,
+        torchao.quantization.Int4WeightOnlyConfig,
+        torchao.quantization.IntxWeightOnlyConfig,
+        torchao.quantization.Int8DynamicActivationIntxWeightConfig,
+        torchao.quantization.ModuleFqnToConfig,
```

Comment:
nit: maybe add `torchao.quantization.FqnToConfig` as well, that's going to replace `ModuleFqnToConfig` I think

Also after the PR, can we turn on safe tensor serialization by default for all non-prototype configs now, or how far are we from that?